src/main/java/pl/gispartner/ResourceManagerApp/api/ResourceController.java

```diff
@@ -0,0 +1,36 @@
+package pl.gispartner.ResourceManagerApp.api;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import pl.gispartner.ResourceManagerApp.model.ResourceDto;
+import pl.gispartner.ResourceManagerApp.service.ResourceService;
+
+@RestController
+@RequestMapping("/api/resource")
+@RequiredArgsConstructor
+public class ResourceController {
+
+    private final ResourceService resourceService;
+
+    @PostMapping
+    public Long saveResource(@RequestBody ResourceDto resourceDto) {
+        return resourceService.saveResource(resourceDto);
+    }
+
+    @DeleteMapping("/{resourceId}")
+    public void deleteResource(@PathVariable("resourceId") Long resourceId) {
+        resourceService.deleteResource(resourceId);
+    }
+
+    @PutMapping("/name/{resourceId}")
```

Comment:
w tej sytuacji jak zlozysz sobie urla to bedziesz mial /api/resource/name/{id} a to nie jest id imienia tylko zasobu. Zamien na /api/resource/{id}/name - w pozostalych miejscach zrob analogicznie tj. dla usera i w polach dla name i jsonData

Reply:
przekazalbym tez usera tutaj w jakis sposob i dodal walidacje czy user jest ownerem. Unikniesz sytuacji kiedy inny uzytkownik zmodyfikuje twoj zasob